src/Shells/StrixMusic.Shells.ZuneDesktop/Controls/Views/Items/ZuneTrackItem.cs

```diff
@@ -17,228 +17,11 @@ namespace StrixMusic.Shells.ZuneDesktop.Controls.Views.Items
     /// </summary>
     public class ZuneTrackItem : TrackItem
     {
-        private bool _isSelected = false;
-
-        /// <summary>
-        /// The top most grid around the track item.
-        /// </summary>
-        public Grid? PART_MainGrid { get; private set; }
-
-        /// <summary>
-        /// GradientStop for the artist column.
-        /// </summary>
-        public GradientStop? PART_GradientStopArtist { get; private set; }
-
-        /// <summary>
-        /// GradientStop for the track column.
-        /// </summary>
-        public GradientStop? PART_GradientStopTrack { get; private set; }
-
-        /// <summary>
-        /// Holds the list of artists.
-        /// </summary>
-        public string ArtistString
-        {
-            get { return (string)GetValue(ArtistStringProperty); }
-            set { SetValue(ArtistStringProperty, value); }
-        }
-
-        /// <summary>
-        /// Dependency property for <see cref="ArtistString" />.
-        /// </summary>
-        public static readonly DependencyProperty ArtistStringProperty =
-            DependencyProperty.Register(nameof(ArtistString), typeof(string), typeof(ZuneTrackItem), new PropertyMetadata(null));
-
-        /// <summary>
-        /// Holds the current state of the zune <see cref="ITrackCollectionViewModel"/>.
-        /// </summary>
-        public ITrackCollectionViewModel ParentCollection
-        {
-            get { return (ITrackCollectionViewModel)GetValue(ParentCollectionProperty); }
-            set { SetValue(ParentCollectionProperty, value); }
-        }
-
-        /// <summary>
-        /// Dependency property for <ses cref="ITrackCollectionViewModel" />.
-        /// </summary>
-        public static readonly DependencyProperty ParentCollectionProperty =
-            DependencyProperty.Register(nameof(ParentCollection), typeof(ITrackCollectionViewModel), typeof(ZuneTrackItem), new PropertyMetadata(null));
-
-
-        /// <summary>
-        /// Holds the current state of the zune <see cref="ParentAlbumArtistCollection"/>.
-        /// </summary>
-        public IAlbumCollectionViewModel ParentAlbumArtistCollection
-        {
-            get { return (IAlbumCollectionViewModel)GetValue(ParentAlbumArtistCollectionProperty); }
-            set { SetValue(ParentAlbumArtistCollectionProperty, value); }
-        }
-
-        /// <summary>
-        /// Dependency property for <ses cref="IAlbumCollectionViewModel" />.
-        /// </summary>
-        public static readonly DependencyProperty ParentAlbumArtistCollectionProperty =
-            DependencyProperty.Register(nameof(ParentAlbumArtistCollection), typeof(IAlbumCollectionViewModel), typeof(ZuneTrackItem), new PropertyMetadata(null));
-
-
         /// <summary>
         /// Creates a new instance of <see cref="ZuneTrackItem"/>.
         /// </summary>
         public ZuneTrackItem()
```

Comment:
This will be needed in future.

Reply:
It's still here. I'm not sure what this comment is for, does something need to be changed?